test/fixtures/rules/std_rule_cases/L044.yml

```diff
@@ -514,3 +514,13 @@ test_snowflake_delete_cte:
   configs:
     core:
       dialect: snowflake
+
+test_pass_exasol_values_clause_cte:
+  pass_str: |
+    WITH txt AS (
+        VALUES (1)
```

Comment:
Can we have an example with multiple values?

Reply:
Can do. I will say, though, the new code doesn't look at that part at all, AFAIK.